docs/section.md

```diff
@@ -116,21 +116,21 @@ The most usual use case is a test step with a button click action.
 
 The section element code declaration containing the timeout attribute:
 
-> StorefrontSigninSection.xml
+> StorefrontCustomerSignInPopupFormSection.xml
 
 ```xml
 ...
-<element name="signIn" type="button" selector="#signIn" timeout="30"/>
+<element name="signIn" type="button" selector="#send2" timeout="30"/>
```

Comment:
Is this change necessary?

Reply:
updated it to match current value of `signIn` in `StorefrontCustomerSignInPopupFormSection`